### DISCUSSION TITLE
Economy Add-ons Ideas.

### DISCUSSION THREAD

**Discussion by ShikhaliyevRustam** [ParadiseSS13/Paradise#19804] [Documents]
Hello! This is my 2nd post and the last one about the viro; some part of it made it into the game so let's see if we can do it two times!

I'm currently liking the economy reworks and the discussion around this topic and how it should be handled. So let us talk about how we value money right now. We value what we can buy with this currency which is food or ciggies from vendors or using it in cargo for specific crates. Now food from vendors is necessary for the crew, while the rest are amenities.

**Food idea.**
I heard talks of food being more expensive to make money more valuable. I propose you brand the food since we live in a megacorp society. Things like Space Twinkie should not be re-branded since it is in itself a brand; however, things like canned soup could be renamed to something else to cause the brand to be a main driving force for price.

**Amenity idea.**
We should add more amenities to the game. One good example of mass spending and where the money actually matters is when a trade shuttle arrives on the station, then players are lining up to spend their money, and the real value of money can be observed. I advocate that we have a lesser form of that shuttle that will arrive at a station, regardless of what alert the station is on. The shuttle should only contain vending machines that cannot be tampered with, meaning they cannot be disassembled, and destroying them drops nothing. It is like those autonomous grocery stores that Amazon tried to do in Sweden. The items in the machines should be low-priced items from the trade shuttle and some other items which are not worth that much in the game, i.e., night vision med huds. It should also have a short time as well before it departs, causing a FOMO effect on the crew where they start spending their money since, at that point, their money has value. You could use that one vending machine below the PDA on cyberiad in cargo as the UI for the new ones on this new shuttle. This could be an event that triggers if the station is on red when the trade shuttle event triggers.

I'm thrilled we are having a lot of discussion on how to make money more valuable, and I hope at least a part of my idea gets incorporated into the game.

**Comment by GDNgit** [comment]
I like the amenity idea, but it really shouldn't come on higher alerts. The entire point of the trader shuttles is to punish people for keeping the station on a high alert even after a situation is resolved.

**Reply by ShikhaliyevRustam** [reply]
I think, this is a fair point it could put crew into danger, maybe we can add a single vendor into the station and that would serve as a better example and the fomo would be it just shutting down after a certain time or it has a limited supply. The ball is in your court.
